### DISCUSSION TITLE
various problems flashing Davinci 1.0A with reptier 0.92

### DISCUSSION THREAD

**Discussion by stuiven** [luc-github/Repetier-Firmware-0.92#379] [F.A.Q]
hello together, i hope you could help me, because the voltivo forum isnt available. 

i tried to falsh my printer Davinci 1.0A with repetier 0.92 as recommended. 
I downloaded arduino 1.8.0 installed the board with board management system 1.6.8, then i updatet variant.ccp and USCcore.ccp in the new places for them C:\Users\Username\AppData\Local\Arduino15\packages\arduino\hardware\sam\1.6.8

Filesize:
variant.ccp 27,9 KB (28.573 Bytes)
USBCore.ccp 23,5 KB (24.086 Bytes)

I edit the configuration.h with 1-1-0 Davinci Printer with J37 and no fan.

I have seen an older post which said the filesize should be higher, but this is ecately the file size i could download in your space.

I don`t get the printer working, after flashing the firmware i get the disconnection sound from windows, but i can flash i immediately new with arduino, so it does not disconnect. 
Sometimes i heard the sound of rebooting and the printer lighted up , but hte screen wasent working, i showd light black bars, but i could move axis with repetier. i already checked the displaycable and changed it with a new one, which doesnt help.
before flashing the printer i get the normal black bars with good color, so i think the display itself is ok and the problem is in the firmware. 

Do you have any idea whats wrong with flashing the new firmware? 

Best regards Steff
Could you help maybe help me with installing repetier on the printer?

**Comment by luc-github** [comment]
did you power off power on the printer after flash ? the screen may need a hard reset

**Reply by stuiven** [reply]
Yes i did various times, bzt the screen is all the time the same.

you can see very light black bars which seem to vary a bit in the saturation of the black colour, but the screen doesnt work properly.
Before the flash they are as normal without any firmware.

**Comment by luc-github** [comment]
try to flash binary from release to check  if it is due to some mistake
https://github.com/luc-github/Repetier-Firmware-4-Davinci/releases

**Reply by stuiven** [reply]
Thank you for your answer. I tried to upload it and the upload worked well. After the upload it seems to be the same as before.

i used the 1.0A firmware from 15 Mar 2018, as in the top post.

my setup is a 1.0A (blue sidepanals) 
Jumper j37
No fan for partcooling only one fan for cooling the hotend.


Have you any other ideas? 
For me the screen seems to work fine, the black bars are fading after the firmware upload.
![B2977BC1-DF4E-486D-A3C8-313D93C86338](https://user-images.githubusercontent.com/82287382/114384033-80976980-9b8e-11eb-91cc-32f483d209f0.jpeg)

**Reply by luc-github** [reply]
>After the upload it seems to be the same as before.

Sorry do you means the lcd has black bars ? or it is fine ?
which binary did you used (Full name)?
